packages/backend/src/clients/Slack/SlackClient.ts

```diff
@@ -22,7 +22,8 @@ type SlackClientArguments = {
     lightdashConfig: LightdashConfig;
 };
 
-const CACHE_TIME = 1000 * 60 * 10; // 10 minutes
+const DEFAULT_CACHE_TIME = 1000 * 60 * 10; // 10 minutes
+const MAX_CHANNELS_LIMIT = 100000;
```

Comment:
Considering we can now use search, I'd suggest to reduce this to 1000 , what do you think ?

Reply:
Hi, sure we can do that, from my experience 100k is around the edge of causing that maximum call stack size exceed error, but i want make sure before search is fully tested and ready the users with small workspace is not impacted, for small workspace it would just using the `MultiSelect` build-in search to handle this.